### PR TITLE
Cognitive complexity and simplification 2

### PR DESCRIPTION
**Pull request by helpau**

### CODE REVIEW

**Comment by helpau** [comment]
@Unlucky-Life please allow squah merge in repo settings. Add-on config files are not in gitignore, so i can't rebase commit the easy way.

File: src/Ankimon/__init__.py [hunk 1]
```diff
@@ -446,21 +446,18 @@ def addon_config_editor_will_display_json(text: str) -> str:
...
-            if response.status_code == 200:
-                return True
-        except requests.ConnectionError:
-            # Connection error means no internet connectivity
-            return False
-    online_connectivity = test_online_connectivity()
-except:
-    online_connectivity = False
+        # Check if the response status code is 200 (OK)
+        if response.status_code == 200:
```
Comment (helpau) [inline src/Ankimon/__init__.py:455]: What if response code is not 200? Probably there is a bug

File: src/Ankimon/__init__.py [hunk 2]
```diff
@@ -6755,6 +6755,7 @@ def eventFilter(self, obj, event):
                     test_window.display_first_start_up()
                 else:
                     test_window.open_dynamic_window()
+        return False
```
Comment (helpau) [inline src/Ankimon/__init__.py:6758]: Bug when Win button pressed, created in #167 , bugfix here

**Comment by Unlucky-Life** [comment]
> @Unlucky-Life please allow squah merge in repo settings. Add-on config files are not in gitignore, so i can't rebase commit the easy way.

image
=> its actually already in the settings

**Pull request merged**